Flutter/datagrid/filtering.md

```diff
@@ -1325,4 +1325,68 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Show the column header icons when hovering the header cell

```

Comment:
Move this topic inside the Column Types,

https://help.syncfusion.com/flutter/datagrid/column-types

Reply:
@ashok-kuvaraja - added to column Types